src/components/calcite-tree-item/calcite-tree-item.tsx

```diff
@@ -418,4 +418,11 @@ export class CalciteTreeItem {
       return;
     }
   };
+
+  private isSelectionMulti(): boolean {
```

Comment:
Looking for a better name for this method. We can use `isSelectionMultiOrMultiChildren` which is a bit long .Any suggestions?

Reply:
`isSelectionMultiLike`? Your suggestion also works since it's descriptive enough. It's private too, so we can refactor if necessary.